src/main/resources/.env

```
@@ -0,0 +1,3 @@
+export AZURE_CLIENT_ID=e786f550-574c-4265-8a86-392f6ca48e29
+export AZURE_CLIENT_SECRET=3fa6cfbe-5426-4e61-bbe4-1eef26f25ae8
+export AZURE_TENANT_ID=65acc86a-7d1c-4792-ac71-b5d8f9f969b6
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)